src/modifiers.rs

```diff
@@ -19,7 +19,8 @@ impl StatusModifier for StateFromPeopleNowPresent {
         let people_now_present: Option<u64> = status
             .sensors
             .as_ref()
-            .map(|sensors| sensors.people_now_present[0].value);
+            .and_then(|sensors: &api::Sensors| sensors.people_now_present.first())
+            .map(|sensor: &api::PeopleNowPresentSensor| sensor.value);
```

Comment:
Should we log if no sensor is present? It probably deserves a warning, since if one adds the modifier one would expect the sensor to be present.

Reply:
This can also happen if a sensor is configured, but when the database is still empty. This is already being logged, so I don't think we need to do anything else.

Reply:
OK.